Testing/SystemTests/tests/framework/CrystalFieldPythonInterface.py

```diff
@@ -123,7 +123,7 @@ def runTest(self):
         # ---------------------------
 
         cf = CrystalField('Ce', 'C2v', B20=0.37737, B22=3.9770, B40=-0.031787, B42=-0.11611, B44=-0.12544,
```

Comment:
The tests were failing due to lists of mixed types no longer being supported. I admit this makes me a bit uncomfortable. @DavidFair do you remember how difficult it would be to avoid this?

Reply:
In this case the breakage was intentional, as the implementation guesses if we have a float or a integer type.
The problem starts when we get a list of mixed lists, as the PropertyManager needs us to pick between floats or ints.

For this reason we now explicitly error out explaining to the user we either need a list of floats, or a list of ints. But we cannot take a mixed list due to the precision loss / chance of errors associated.